yapapi/executor/__init__.py

```diff
@@ -451,26 +468,33 @@ async def accept_payment_for_agreement(
             events.PaymentAccepted(agr_id=agreement_id, inv_id=inv.invoice_id, amount=inv.amount)
         )
 
-    def approve_agreement_payments(self, agreement_id):
+    def approve_debit_notes_for_agreement(self, agreement_id):
+        """Add given agreement to the set of agreements for which debit notes should be accepted."""
         self._agreements_accepting_debit_notes.add(agreement_id)
 
     def add_job(self, job: "Job"):
+        """Register a job with this engine."""
         self._jobs.add(job)
 
     @staticmethod
     def finalize_job(job: "Job"):
+        """Mark a job as finished."""
         job.finished.set()
 
     @dataclass
     class PaymentDecoration(DemandDecorator):
```

Comment:
Shouldn't this be named `PaymentDecorator`? :thinking:

Reply:
![](https://github.githubassets.com/images/icons/emoji/unicode/1f914.png)

Reply:
I guess it should

Reply:
fixed in 761f820